auslib/admin/views/forms.py

```diff
@@ -155,6 +158,33 @@ class ExistingPermissionForm(DbEditableForm):
     options = JSONStringField(None, 'Options')
 
 
+class ScheduledChangeNewPermissionForm(ScheduledChangeTimeForm, NewPermissionForm):
+    permission = StringField('Permission', validators=[Length(0, 50), InputRequired()])
+    username = StringField('Username', validators=[Length(0, 100), InputRequired()])
+    change_type = SelectField("Change Type", choices=[('insert', 'insert'), ('update', 'update'), ('delete'), ('delete')])
```

Comment:
Yeah, I struggled with what to do here. There's so many different combinations of required/not required/not allowed for PKs, non-PK base table columns, scheduled changes columns, data_version/sc_data_version that I opted to just be more explicit and verbose.

In the medium-to-long term, moving towards Swagger may help make this better. We should be able to separate "definition and validation of a field" from "required/not required/not allowed".